.github/workflows/deploy-public-build.yml

```diff
@@ -43,8 +44,9 @@ jobs:
           path: ./dist
 
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v3
+        uses: JamesIves/github-pages-deploy-action@v4
```

Comment:
Replaced because `peaceiris/actions-gh-pages` does not support excluding subfolders from deletion/cleaning